cg/meta/transfer/utils.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from pathlib import Path
+
+from cg.utils.checksum.checksum import is_md5sum_correct
+
+LOG = logging.getLogger(__name__)
+
+
+def get_all_fastq_from_folder(sample_folder: Path) -> list[Path]:
+    """Returns a list of all fastq.gz files in given folder."""
+    all_fastqs: list[Path] = []
+    for leaf in sample_folder.glob("*fastq.gz"):
+        abs_path: Path = sample_folder.joinpath(leaf)
+        LOG.info(f"Found file {str(abs_path)} inside folder {sample_folder}")
+        all_fastqs.append(abs_path)
+    return all_fastqs
```

Comment:
We have a `files.py` in `cg/utils` we should add file handling utility functions in there I think and re-use them as much as possible. `get_files_matching_pattern` It is called I believe